brave/browser/brave_javascript_dialog_manager.cc

```diff
@@ -189,6 +189,7 @@ void BraveJavaScriptDialogManager::RunJavaScriptDialog(
 
 void BraveJavaScriptDialogManager::RunBeforeUnloadDialog(
     content::WebContents* web_contents,
+    content::RenderFrameHost* render_frame_host,
```

Comment:
I think we need to add a check here for the main frame and include true/false for it in the emit

Reply:
@bridiver the change was only done upstream to measure the schemes of pages showing beforeunload dialogs.